cloudera-integration/csd/descriptor/service.sdl

```diff
@@ -103,6 +103,81 @@
         ]
       }
     },
+    {
+      "name": "LOG_COLLECTOR_KAFKA",
+      "label": "Log Collector Kafka",
+      "pluralLabel": "Log Collectors Kafka",
+      "inExpressWizard": true,
+      "jvmBased" : "true",
+      "parameters": [
+        {
+          "name": "topic",
+          "label": "Kafka topic",
+          "description": "Log collector Kafka Topic",
```

Comment:
A longer description here would be helpful. What is the topic for? Can you have more than one? What does the log collector do with the topic?

Reply:
@afoerster how does this sound?
`Kafka topic name for log messages sent to logstash and picked up via Log Collector on kafka mode. Takes in only one topic.`

Reply:
rewrote description to 

> Kafka topic name for log messages sent to logstash and picked up via Log Collector in kafka mode. Takes as an argument a single topic.